Makefile

```diff
@@ -600,15 +596,14 @@ copy-go-binaries-to-image-dir:
 	cp bin/linux_$(GOARCH)/central image/bin/central
 ifdef CI
 	cp bin/linux_amd64/roxctl image/bin/roxctl-linux-amd64
+	cp bin/linux_arm64/roxctl image/bin/roxctl-linux-arm64
```

Comment:
WDYT about having a makefile variable contain a list of all roxctl platforms? this would make it easier to keep things consistent, especially with the CI/non-CI flavors.